edc-extensions/bpn-validation/bpn-validation-core/src/test/java/org/eclipse/tractusx/edc/validation/businesspartner/functions/BusinessPartnerGroupFunctionTest.java

```diff
@@ -146,24 +144,15 @@ private static class ValidOperatorProvider implements ArgumentsProvider {
         @Override
         public Stream<? extends Arguments> provideArguments(ExtensionContext extensionContext) {
             return Stream.of(
-                    Arguments.of("Matching groups", EQ, List.of(TEST_GROUP_1, TEST_GROUP_2), true),
-                    Arguments.of("Disjoint groups", EQ, List.of("test-group", "different-group"), false),
-                    Arguments.of("Overlapping groups", EQ, List.of("different-group"), false),
-
-                    Arguments.of("Disjoint groups", NEQ, List.of("different-group", "another-different-group"), true),
-                    Arguments.of("Overlapping groups", NEQ, List.of(TEST_GROUP_1, "different-group"), true),
-                    Arguments.of("Matching groups", NEQ, List.of(TEST_GROUP_1, TEST_GROUP_2), false),
-                    Arguments.of("Empty groups", NEQ, List.of(), true),
```

Comment:
also if it is deprecated I would keep testing for "eq" and "neq", that will be removed at the time the operators will be removed.

Reply:
thank you, you're right, changed in [here](https://github.com/eclipse-tractusx/tractusx-edc/pull/1673/commits/2bf5913849973a84a251f9b407e5a8599727152b).